eip_config/config/application.properties

```diff
@@ -0,0 +1,168 @@
+openmrs.db.host=${OPENMRS_DB_HOST}
+
+# The OpenMRS MySQL database port
+openmrs.db.port=${OPENMRS_DB_PORT}
+
+# The OpenMRS MySQL database name
+openmrs.db.name=${OPENMRS_DB_NAME}
+# *********************** General EIP sender properties ****************************************************************
+#
+
+# Maximum size of each batch of source records to be read by debezium from the binlog files, maps to the
+## debezium MySQL connector property named max.batch.size
+#https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-property-max-batch-size
+debezium.reader.maxBatchSize=2048
+
+#Enabled Oauth when set to true
+oauth.enabled=false
+
+#The client Id of the account
+oauth.access.token.uri=${KEYCLOAK_URL}/auth/realms/ozone/protocol/openid-connect/token
+
+#The client Id of the account to use to authenticate
+oauth.client.id=eip
+
+#The client secret of the account to use to authenticate
+oauth.client.secret=${EIP_CLIENT_SECRET}
+
+#Authentication scope, can be multiple values separated by commas
+oauth.client.scope=email
+
+eip.home=/eip-home
+
+# Camel endpoints that need to be notified of DB events
+db-event.destinations=direct:odoo-event-listener,direct:outbound-toLims
+
+# A comma separated list of database tables names to watch for changes
+eip.watchedTables=orders,test_order
```

Comment:
Could you make sure to watch the right tables?
As discussed the other day I believe Odoo routes need to watch more than those 2 tables: See https://github.com/mekomsolutions/eip-routes-odoo/blob/87ea7b58561f2fa961cc347eac7f019e0e5c1e3f/src/test/resources/application.properties#L6